FRPs/proposal-antitrust.md

```diff
@@ -0,0 +1,44 @@
+Increasing Competition Using Decentralized Technologies
+
+(1) Overview
+
+The Decentralization Research Center (DRC) is excited to submit the research proposal below. We are a nonprofit organization that supports nonpartisan decentralization research and acts as a hub for critical stakeholders. Our mission is to encourage the equitable ownership, governance, and regulation of emerging technologies that impact our everyday lives.
+
+In our proposed research, we aim to investigate how decentralization, especially as enabled by blockchain technology, can impact competition and offer solutions to monopolistic tendencies in various sectors. Central to this investigation is the role of MEV in fostering competitive environments and their implications for future antitrust policies and regulations. The study is inspired by some of our previous research, the paper we co-authored for the Harvard Kennedy School on equitable ownership and governance, and discussions with MEV builders.
```

Comment:
Suggesting that we frame anything regarding MEV more through the lens that "the body of research and development into MEV has led to tools and protocols that can serve purposes beyond public blockchains and can be applied to sectors where privilege actors maintain dominate or monopolistic tendencies."

Rather, we should talk about MEV directly. Its more about how our research can be applied in other areas.

Reply:
Perfect!